### PR TITLE
Added how-to article transfer a project to another organisation

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: transfer-organisation.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. _how-to-transfer-organisation:
+
+How to transfer a project to another organisation
+=================================================
+
+Projects can be moved to different organisations you belong to. 
+
+To transfer a project to another organisation, first you must cancel its subscription via the Subscription view. A project cannot be transferred while it has a subscription applied.
```
Comment (evildmp) [inline transfer-organisation.rst]: Check column length please

File: transfer-organisation.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. note:: For projects with business subscription only: 
+
+    if you don't want to interrupt the subscription, please contact Divio support and we can transfer the project
```
Comment (evildmp) [inline transfer-organisation.rst]: Capitalisation - if.

File: transfer-organisation.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    without any downtime for you.
+
+Once the subscription has been cancelled, open the options menu in the project's dashboard and select Transfer. 
+
+.. image:: /images/options-menu-project-view.png
```
Comment (evildmp) [inline transfer-organisation.rst:24]: Use both images:

```
.. image:: /images/options-menu-organisations-view.png
   :alt: 'options menu'
   :width: 685

or in the project view:

.. image:: /images/options-menu-project-view.png
   :alt: 'options menu'
   :width: 690
```

**Review by evildmp — CHANGES_REQUESTED** [review]

File: transfer-organisation.rst (hunk 2 above)
Comment (mebverse) [reply transfer-organisation.rst]: Done

File: transfer-organisation.rst (hunk 3 above)
Comment (mebverse) [reply transfer-organisation.rst:24]: Done

File: transfer-organisation.rst (hunk 1 above)
Comment (mebverse) [reply transfer-organisation.rst]: Done

File: transfer-organisation.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+How to transfer a project to another organisation
+=================================================
+
+Projects can be moved to different organisations you belong to. 
+
+To transfer a project to another organisation, first you must cancel its subscription via the Subscription view. A
+project cannot be transferred while it has a subscription applied.
+
+.. note:: For projects with business subscription only:
```
Comment (evildmp) [inline transfer-organisation.rst]: _Business subscriptions_

Comment (mebverse) [reply transfer-organisation.rst]: Done

**Pull request merged**